### PR TITLE
Add :BracketRanking

### PR DESCRIPTION
**Pull request by SmedbergM**

### CODE REVIEW

File: athlinks-ontology.ttl [hunk 1]
```diff
@@ -111,9 +111,9 @@ dc:date rdf:type owl:AnnotationProperty .
          rdfs:subClassOf :AthleticEventRole ;
          dc:creator "nextdude"^^xsd:string ;
          dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime ;
-         rdfs:comment "A role that a person or sports team plays when they participate in an athletic event."@en ;
+         rdfs:comment "A role that an entry (such as a person or sports team) plays when they participate in an athletic event."@en ;
          rdfs:label "Athletic Event Competitor"@en .
-         # members:
+         # expected predicates:
          # schema:competitor -> individual (e.g. schema:Person) or team (e.g. schema:SportsTeam)
          # schema:sportsEvent -> athlinks:AthleticEvent
```
Comment (nextdude) [inline athlinks-ontology.ttl:118]: This should be schema:event I think

File: athlinks-ontology.ttl [hunk 2]
```diff
@@ -367,10 +367,30 @@ dc:date rdf:type owl:AnnotationProperty .
             rdfs:label "Swim Action"@en .
 
 
-###  http://schema.org/GenderType
-schema:GenderType dc:creator "nextdude"^^xsd:string ;
-                  dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime .
 
+###  http://adp.athlinks.com/schema/BracketRanking
+:BracketRanking rdf:type owl:Class ;
```
Comment (nextdude) [inline athlinks-ontology.ttl]: @SmedbergM There are existing classes for AthleteResult (aka bracketranking), AthleteResultList (aka leaderboard) and AthleteCohort (aka bracket). Take a look and see if those cover what you need. They still need properties.

File: athlinks-ontology.ttl [hunk 3]
```diff
@@ -367,10 +367,30 @@ dc:date rdf:type owl:AnnotationProperty .
...
+
+###  http://adp.athlinks.com/schema/rank
+:rank a owl:DatatypeProperty ;
+  rdfs:domain :BracketRanking ;
+  rdfs:range schema:Int ;
+  rdfs:comment "The numeric rank of an individual in a ranking."@en ;
+  rdfs:label "rank"@en .
+
+###  http://adp.athlinks.com/schema/ActivityType
 :ActivityType
```
Comment (nextdude) [inline athlinks-ontology.ttl:434]: I really want to get rid of `ActivityType`. It was a hack to support the early work in the `adp-api`, but its not necessary. I envision our event types to be subclasses of `ExerciseAction` and we can use the `potentialAction` property on `Thing` to support describing the type of events, races, and results.

**Review by nextdude — CHANGES_REQUESTED** [review]
I'm doing some work on the schema this weekend and will share a PR with you on refactoring `ActivityType`. In the meantime, would be interested to know if we can get away without adding another custom class for `BracketRanking`.

File: athlinks-ontology.ttl (hunk 1 above)
Comment (SmedbergM) [reply athlinks-ontology.ttl:118]: If it's an *Athletic* event competitor, then it seems reasonable to use the sports-specific predicate. But I don't care to argue the point too terribly much.

File: athlinks-ontology.ttl (hunk 2 above)
Comment (SmedbergM) [reply athlinks-ontology.ttl]: The reason I didn't use those is that there's supposed to be a many-to-one relationship between entries and AthleteResultList and AthleteCohort. Here, each BracketRanking should have exactly one entry pointing to it.

A single AthleteResult can have several rankings (e.g. if the athlete is in several brackets).

File: athlinks-ontology.ttl [hunk 4]
```diff
@@ -366,11 +374,63 @@ dc:date rdf:type owl:AnnotationProperty .
...
+  rdfs:comment "A predefined rule for associating entries to a cohort."@en ;
+  rdfs:label "Cohort Ranking Type"@en .
+
+:cohortRankingType a owl:ObjectProperty ;
+  rdfs:domain :CohortRanking ;
+  rdfs:range [
+    rdf:type owl:Class ;
+    owl:unionOf(
+      :CohortRankingType
+      schema:string
```
Comment (nextdude) [inline athlinks-ontology.ttl:389]: We should use xsd:string; schema:string doesn't exist in the api.

File: athlinks-ontology.ttl [hunk 5]
```diff
@@ -366,11 +374,63 @@ dc:date rdf:type owl:AnnotationProperty .
...
+:cohortRanking a owl:ObjectProperty ;
+  rdfs:domain :AthleteResult ;
+  rdfs:range :CohortRanking ;
+  rdfs:comment "A ranking of the individual result within a cohort."@en ;
+  rdfs:label "cohort ranking"@en .
+
+###  http://adp.athlinks.com/schema/rank
+:rank a owl:DatatypeProperty ;
+  rdfs:domain :CohortRanking ;
+  rdfs:range schema:Int ;
```
Comment (nextdude) [inline athlinks-ontology.ttl:429]: use xsd:integer

Comment (warrencj) [reply athlinks-ontology.ttl:429]: Fixing

File: athlinks-ontology.ttl (hunk 4 above)
Comment (warrencj) [reply athlinks-ontology.ttl:389]: Fixing

**Pull request merged**